spectral/ruleset.yaml

```diff
@@ -9,6 +9,21 @@ functions:
 rules:
   openapi-tags: off
   operation-tags: off
+  # openapi spec rules, that return errors
+  oas3-valid-schema-example: off
```

Comment:
-1 to disable it. That should be trivial fix on any API we have.

Reply:
ok so keep those in there and fix up the KFM etc?